StyleSheet.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blue;
+}
```

Comment:
- [ ] Good to have your CSS file :100: But from the project requirement, your CSS file should include one class with styles for your header. But you style the `Hello Microveerse` using the tag name `h1`, please kindly style it using it class name instead of tag name :smile: